egui_web/src/http.rs

```diff
@@ -24,6 +24,10 @@ async fn fetch_jsvalue(request: &Request) -> Result<Response, JsValue> {
     opts.method(method);
     opts.mode(web_sys::RequestMode::Cors);
 
+    if method == &"POST".to_string() {
```

Comment:
```suggestion
    if !body.is_empty() {
```

`POST` is not the only verb that wants to send data (there is also `PUT`, and maybe others).